vendoropts.go

```diff
@@ -0,0 +1,54 @@
+package dhcp6
+
+import (
+	"encoding/binary"
+	"io"
+)
+
+// This option is used by clients and servers to exchange
+// vendor-specific information.
+type VendorOpt struct {
+	// EnterpriseNumber specifies an IANA-assigned vendor Private Enterprise
+	// Number.
+	EnterpriseNumber uint32
+
+	// An opaque object of option-len octets,
+	// interpreted by vendor-specific code on the
+	// clients and servers
+	Options Options
+}
+
+// MarshalBinary implements VendorOpt, and allocates a byte slice containing the data
```

Comment:
Do you prefer to fix the comment from previous PR by yourself or do you prefer that I will do the jobs (only tomorrow)